cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/preview/PreviewHttpHandler.java

```diff
@@ -106,12 +106,12 @@
 
   @Inject
   PreviewHttpHandler(CConfiguration cConf, Configuration hConf, DiscoveryService discoveryService,
-                            DatasetFramework datasetFramework, PreferencesStore preferencesStore,
-                            SecureStore secureStore, TransactionManager transactionManager,
-                            ArtifactRepository artifactRepository, ArtifactStore artifactStore,
-                            AuthorizerInstantiator authorizerInstantiator, StreamAdmin streamAdmin,
-                            StreamCoordinatorClient streamCoordinatorClient, PrivilegesManager privilegesManager,
-                            AuthorizationEnforcer authorizationEnforcer) {
+                     @Named(DataSetsModules.BASE_DATASET_FRAMEWORK) DatasetFramework datasetFramework,
```

Comment:
So this dataset framework won't have authorization? We are not going to do authorization for preview?